cypress/e2e/supervision/orders/bonds/edit-bond.cy.js

```diff
@@ -0,0 +1,33 @@
+beforeEach(() => {
+  cy.loginAs("Case Manager");
+  cy.createClient()
+    .withOrder()
+    .withBond();
+});
+
+describe(
+  "Edit bond",
+  () => {
+    Cypress._.times(50, () => {
+      it("Triggering the validation summary when editing a bond in supervision", () => {
+        cy.get("@client").then(({id: clientId}) => {
+          cy.visit(
+            `/supervision/#/clients/${clientId}`
+          );
+        });
+
+        cy.get(".TABS_ORDERS").click();
+        cy.get("#tab-order-list-in-page-notification").should("contain", "Please wait...");
+        cy.get("#tab-order-list-in-page-notification").should("not.contain", "Please wait...");
+
+        cy.get(".edit-bond-button").click();
+        cy.get('#requiredBondAmount').clear();
+        cy.get('#requiredBondAmount').type("999999999999")
+        cy.contains("button", "Save & exit").click();
+        cy.get(".validation-summary")
+          .and("contain.text", "There is a problem")
+          .and("contain.text", "Required bond amount - The input is not less than '10000000000'");
+      });
+    });
+  }
+);
```

Comment:
It doesn't look like there's a test for the original 'the bond should look as expected', is this shown anywhere else?

Reply:
Yeah I thought it was this file Dispense bond

Reply:
So I think dispense bond shows that the bond loads and you can dispense it but the original Sirius test checks that you can edit it and then that info gets displayed. Is it possible to test that too?